src/ImageSharp/Memory/Allocators/ArrayPoolMemoryAllocator.Buffer{T}.cs

```diff
@@ -53,8 +53,13 @@ public override Span<T> GetSpan()
                 {
                     ThrowObjectDisposedException();
                 }
-
+#if SUPPORTS_CREATESPAN
```

Comment:
This came up during profiling. On NET Core 3.1 We can use a shortcut since we know the length.